bin/auth-api/src/routes/workspace.routes.ts

```diff
@@ -424,6 +404,20 @@ router.get("/workspaces/:workspaceId/go", async (ctx) => {
   ctx.redirect(redirectUrl);
 });
 
+router.post("/workspaces/:workspaceId/createAutomationToken", async (ctx) => {
+  const { authUser, workspace } = await authorizeWorkspaceRoute(ctx, RoleType.OWNER);
+
+  const token = createSdfAuthToken({
+    userId: authUser.id,
+    workspaceId: workspace.id,
+    role: "automation",
+  }, {
+    expiresIn: "1 day",
```

Comment:
it's been a while since I was in JWT spec land - does this add the `exp` claim to the token? I would have expected that to get automatically enforced by SDF's middleware but I could be wayyy off here.

Reply:
I would have too; looking at the token at `https://jwt.io` it appears correct, but I'll definitely be revisiting this.